jsons/1.8-rel.json

```diff
@@ -40,6 +40,13 @@
       "serverreq":true
     },
     {
+      "name": "jline:jline:2.13",
+      "url" : "http://files.minecraftforge.net/maven/",
+      "checksums" : [ "TODO", "TODO" ],
```

Comment:
You will need to upload [jline-2.13](http://search.maven.org/remotecontent?filepath=jline/jline/2.13/jline-2.13.jar) to the files server before this can be merged.